cities/laupheim.json

```diff
@@ -0,0 +1,199 @@
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Fastenmarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": "2017 Apr 06 08:00-18:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88331,
+                    48.22962
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Pfingstmarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": "2017 Jun 01 08:00-18:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88760,
+                    48.23057
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Rosenmarkt",
+                "location": "Schloss Großlaupheim",
+                "opening_hours": "2017 Jun 17 09:00-18:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88348,
+                    48.22973
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Jakobimarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": "2017 Jul 20 08:00-18:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88301,
+                    48.22944
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Kirchweihmarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": "2017 Oct 12 08:00-18:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88927,
+                    48.22782
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Kaltblutpferdemarkt:",
+                "location": "Festplatz Laupheim",
+                "opening_hours": "2017 Oct 12 08:00-16:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88285,
+                    48.22937
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Weihnachtsmarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": null,
+                "opening_hours_unclassified": "2017 Dez 14 08:00-18:00"
```

Comment:
Your latest changes look good. I just noticed one market shows up as "unclassified". Is this on purpose or by mistake?